### PR TITLE
feat: add clone repos script

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR introduces a script to handle the cloning of repos into our EFS. This is meant as a one-off script (or a script that is run very infrequently) to populate the EFS with the contents of all repos - as it does not need to be run on every deploy, we've opted to place this in the tooling repo instead of the CMS repos.

Note that this script also relies on the pre-setup of ssh credentials within the EC2 instance in order to be able to clone private repos using simple-git - this is resolved by https://github.com/isomerpages/isomercms-backend/pull/877

Resolves IS-338.

### CODE REVIEW

**Review by QiluXie — CHANGES_REQUESTED** [review]

File: clone-repos/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        continue
+      }
+      console.log(`Cloning ${repo.name}`)
+      const repoUrl = `git@github.com:${orgName}/${repo.name}.git`
+      await cloneRepository(repoUrl, repoPath)
+    }
+  }
+}
+
+cloneRepos("isomerpages");
```
Comment (QiluXie) [inline clone-repos/index.js:84]: nit and non-blocking:
is it better to also handle the promise rejection for this top level async call, print out the error, and exit with a proper error code?
Something like:
```js
(async function() {
    try {
        await cloneRepos("isomerpages");
    } catch (err) {
        console.log(err); 
        process.exit(1);
    }
})();
```

File: clone-repos/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  "recommender-train",
+  "editor",
+  "ci-test",
+  "infra",
+  "markdown-helper",
+]
+
+const { Octokit } = require("@octokit/rest");
+const fs = require("fs");
+const csv = require("fast-csv");
```
[the remark this answers is not in the document]
Comment (alexanderleegs) [reply clone-repos/index.js]: Removed in https://github.com/isomerpages/isomer-tooling/pull/48/commits/526385b149dc4368413ce565b819294fc852c802!

File: clone-repos/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+async function getRepos(orgName) {
+  try {
+    const repos = await octokit.paginate(octokit.repos.listForOrg, {
+      org: orgName,
+      per_page: 100,
+    });
+    return repos;
+  } catch (err) {
+    console.log(err);
```
[the remark this answers is not in the document]
Comment (alexanderleegs) [reply clone-repos/index.js:49]: Yeah that makes sense, added in https://github.com/isomerpages/isomer-tooling/pull/48/commits/526385b149dc4368413ce565b819294fc852c802

File: clone-repos/index.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+const cloneRepository = async (repositoryUrl, destinationPath) => {
+  try {
+    await git.clone(repositoryUrl, destinationPath);
+  } catch (err) {
+    console.log(err)
+    fs.appendFileSync(
+      "clone-repo.log",
+      `Error cloning repo ${repo}: ${err}\n`
```
[the remark this answers is not in the document]
Comment (alexanderleegs) [reply clone-repos/index.js]: Thanks for catching, fixed in https://github.com/isomerpages/isomer-tooling/pull/48/commits/526385b149dc4368413ce565b819294fc852c802!

File: clone-repos/index.js (hunk 1 above)
Comment (alexanderleegs) [reply clone-repos/index.js]: Changed in https://github.com/isomerpages/isomer-tooling/pull/48/commits/526385b149dc4368413ce565b819294fc852c802!

**Review by QiluXie — APPROVED** [review]
lgtm! Thanks @alexanderleegs for addressing all the comments.

**Pull request merged**